docs/TROUBLESHOOTING.md

```
@@ -0,0 +1,754 @@
+
+If you're still stuck after trying these solutions:
+
+1. **Check Logs:**
+   ```bash
+   docker compose logs --tail=100 api
+   docker compose logs --tail=100 db
+   docker compose logs --tail=100 valkey
+   ```
+
+2. **Search Issues:**
+   - GitHub: https://github.com/markland-bytes/wump/issues
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)